doc/man3/RAND_DRBG_get0_master.pod

```diff
@@ -0,0 +1,81 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_get0_master,
+RAND_DRBG_get0_public,
+RAND_DRBG_get0_private
+- get access to the global RAND_DRBG instances
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ RAND_DRBG *RAND_DRBG_get0_master(void);
+ RAND_DRBG *RAND_DRBG_get0_public(void);
+ RAND_DRBG *RAND_DRBG_get0_private(void);
+
+
+=head1 DESCRIPTION
+
+These functions are part of the OpenSSL L<RAND_DRBG(7)> API.
+
+The default RAND API implementation (RAND_OpenSSL()) utilizes three
+shared DRBG instances which are accessed via the RAND API:
+
+The <public> and <private> DRBG are thread-local instances, which are used
+by RAND_bytes() and RAND_priv_bytes(), respectively.
+The <master> DRBG is a global instance, which is not intended to be used
+directly, but is used internally to reseed the other two instances.
+
+These functions here provide access to the shared DRBG instances.
+
+=head1 RETURN VALUES
+
+RAND_DRBG_get0_master() returns a pointer to the <master> DRBG instance.
+
+RAND_DRBG_get0_public() returns a pointer to the <public> DRBG instance.
```

Comment:
Rather than _pointer to the_, _pointer to a_ since there is a plurality of them now.

Reply:
The formulation _pointer to a_ is a  bit unspecific. How about changing it as follows?

```
RAND_DRBG_get0_master() returns a pointer to the global <master> DRBG instance.

RAND_DRBG_get0_public() returns a pointer to the thread-local <public> DRBG instance.

RAND_DRBG_get0_private() returns a pointer to the thread-local <private> DRBG instance.
```

Note: Originally, I had already formulated it like this, but then I thought the explanation in the DESCRIPTION section would be sufficient:

```
The <public> and <private> DRBG are thread-local instances, which are used by RAND_bytes()
and RAND_priv_bytes(), respectively. The <master> DRBG is a global instance, which is not
intended to be used directly, but is used internally to reseed the other two instances.
 ``` 

Unless you propose an alternative,  I will add it back in.